src/express/commands/cleanup.js

```diff
@@ -64,22 +64,25 @@ async function cleanupServices(doc) {
     }
 
     let cleanupServicesTasks = nodeIps.map(async(ip) => {
+        let command = `sudo mkdir -p /var/lib/heimdall && sudo chmod 777 -R /var/lib/heimdall/ `
+        await runSshCommand(ip, command, maxRetries)
+        
```

Comment:
why is this step needed here at cleanup time?

Reply:
[ae0a58c](https://github.com/maticnetwork/matic-cli/pull/91/commits/ae0a58c399c3212dc253bbeaf033761ff1e6541f)